pouta_blueprints/views/users.py

```diff
@@ -39,7 +39,15 @@ def post(self):
     @marshal_with(user_fields)
     def get(self):
         if g.user.is_admin:
-            return User.query.all()
+            user_query = (
+                User.query
+                .order_by(User.is_admin.desc())
+                .order_by(User.is_group_owner.desc())
+                .order_by(User.is_active)
+                .order_by(User.is_blocked)
+                .order_by(User.is_blocked)
```

Comment:
is this something incredibly genius that i don't understand or a duplicate line?